code/style.css

```diff
@@ -1,4 +1,420 @@
+/*Styling for mobile phone*/
+
+* {
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+}
+
 html {
-  background: #0025ff;
-  color: #fff;
+  color: rgb(43, 36, 23);
+  height: 100%;
+  width: 100vw;
+}
+
+
+body {
+  height: 100%; 
+  background-image: url("./images/material-birchwood.svg");
+}
+
+.container {
+  width: 100%;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.header-container {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  margin: 0;
+  width: auto;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: 60px 246px ;
+  grid-template-rows: 65px 67px;
+  grid-template-areas: 
+  "logo title1 title1"
+  "logo title2 title2";
+}
+
+.logo {
+  width: 60px;
+  grid-area: logo;
+  padding: 15px 0 0 0;
+  margin: 10px 0 0 0px;
+  justify-self: end;
+}
+
+h1.title-name {
+  width: 230px;
+  margin: 15px 0px 0px 10px;
+  grid-area: title1;
+  font-family: 'Love Ya Like A Sister', cursive;
+  font-size: 40px;
+}
+
+h1.title-type {
+  width: 200px;
+  margin: 0px 0px 15px 10px;
+  grid-area: title2;
+  font-family: 'Kaushan Script', cursive;
+  font-size: 46px;
+  color: rgb(8, 106, 233);
+}
+
+  
+.navbar {
+  display: none;
+}
+
+section.big-news {
+  width: 100%;
+  padding: 20px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; 
+}
+
+
+section.small-news {
+ width: 100px;
+ padding: 20px;
+ margin: 0;
+ display: flex;
+ flex-direction: column;
+ align-items: center;
+ justify-content: center;
+}
+
+
+.article {
+  width: 320px;
+  padding: 25px;
+  margin-top: 15px;
+  background-color: white;
+  box-shadow: 5px 7px 10px;
+  font-size: 17px;
+}
+
+figure.big-news {
+  background-color: white;
+  width: 320px;
+  padding: 20px;
+  box-shadow: 4.5px 6px 9px;
+}
```

Comment:
Let's use the whole width, now the cards are quite small:

<img width="677" alt="Skärmavbild 2021-02-02 kl  14 06 29" src="https://user-images.githubusercontent.com/51529201/106604442-f1535180-655f-11eb-83b9-16ca170c0c6b.png">